src/snowflake/snowpark/modin/pandas/utils.py

```diff
@@ -20,23 +20,27 @@
 # Version 2.0.
 
 """Implement utils for pandas component."""
+from __future__ import annotations
 
 from collections.abc import Hashable, Iterator, Sequence
 from types import BuiltinFunctionType
-from typing import Any, Callable, Optional, Union
```

Comment:
Why we need to change this?

Reply:
I'm not sure, I tried to undo those changes but the linter keeps changing it like this

Reply:
The change seems to be due to removal of `Optional` and `Union` type hints in the signatures in this file.